components/FlexibleHeader/tests/unit/FlexibleHeaderTraitColletionDidChangeTests.m

```diff
@@ -22,7 +22,7 @@ @interface FlexibleHeaderTraitCollectionDidChangeTests : XCTestCase
 
 @implementation FlexibleHeaderTraitCollectionDidChangeTests
 
-- (void)testTraitCollectionDidChangeBlockCalledWhenTraitCollectionChanges {
+- (void)testTraitCollectionDidChangeBlockCalledWhenTraitCollectionChangesForFlexibleHeaderView {
```

Comment:
Tip: (Follow-up PR) Please move this to a file reflecting the class's name. Also "ColletionDidChange" is misspelled in this file.

Suggestion: `MDCFlexibleHeaderTests` (or less preferred `MDCFlexibleHeaderTraitCollectionDidChangeTests`).

Reply:
Put these in `FlexibleHeaderViewTraitCollectionTests`, and the controller tests in `FHViewControllerTraitCollectionTests`. Since they test different objects.